<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDslamBoardTablePageWidget</class>
 <widget class="QWidget" name="EditDslamBoardTablePageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="autoUpdateBoardListCheckBox">
     <property name="text">
      <string>Автоматическое получение списка досок</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoNumeringPairCheckBox">
     <property name="text">
      <string>Автоматическая нумерация пар</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="editDslamBoardListTableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="getBoardListAction">
   <property name="text">
    <string>Запрос досок с дслама</string>
   </property>
   <property name="toolTip">
    <string>Запрос досок с дслама</string>
   </property>
  </action>
  <action name="editBoardAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/edit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
   <property name="toolTip">
    <string>Изменить</string>
   </property>
  </action>
  <action name="removeBoardAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/remove.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить</string>
   </property>
  </action>
  <action name="renumeringBoardPairsAction">
   <property name="text">
    <string>Пронумеровать пары</string>
   </property>
   <property name="toolTip">
    <string>Пронумеровать пары</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
